internal/server/handlers/validation/load_balancer_config.go

```diff
@@ -0,0 +1,131 @@
+// everest
+// Copyright (C) 2025 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
```

Comment:
Do we need this validation here as well? It looks like all of this is present in the everest-operator webhook already, so can't we just parse the error from there?

Reply:
Yeah, we could do that ... btw, for things like URL validation and regex, we can use CEL expressions, so we probably don't need any code for it

Reply:
Look for `+kubebuilder:validation:XValidation:rule` tags in this file - https://github.com/percona/everest-operator/blob/main/api/v1alpha1/dataimportjob_types.go